files/templates/alertmanager/alertmanager.yaml

```diff
@@ -24,8 +24,14 @@ route:
     continue: false
   - receiver: null
     matchers:
-    - alertname=~"WorkloadClusterApp.*"
+    - alertname="WorkloadClusterAppNotInstalled"
```

Comment:
We have other alerts for `WorkloadClusterApp`. Should we add a matcher for every alert?